code/go/0chain.net/smartcontract/dbs/event/curator.go

```diff
@@ -0,0 +1,24 @@
+package event
```

Comment:
Can we add test for the same? Either unit or integration would be fine

Reply:
Added.